### ISSUE TITLE
[FEATURE] Toggle option for history explorer.

### ISSUE THREAD

**Issue by EltonChou** [EltonChou/TwitterMediaHarvest#107]
**Is your feature request related to a problem? Please describe.**
Some users might have privacy concern about history explorer, if the device might be used by multi people.

**Describe the solution you'd like**
A toggle option for history explorer.

**Comment by DrSchnagels** [comment]
Hi!
I'd really like to see this added. I'm very concerned by this add-on storing unnecessary history information.
Thanks a lot!

**Comment by Anony400** [comment]
Any update on this being added?

**Comment by UglyShortBaldLoser** [comment]
Yes, please, looking forward to this badly as well. Ideally it would be great to have a single toggle, that would make it so that no history of any kind is stored anywhere. You just download the file and thats it, nothing gets recorded anywhere.
